src/Tracy/Bar.php

```diff
@@ -56,7 +56,7 @@ public function getPanel($id)
 	 */
 	public function render()
 	{
-		$useSession = $this->dispatched && session_status() === PHP_SESSION_ACTIVE;
+		$useSession = $this->dispatched && Debugger::getSession()->isActive();
```

Comment:
Maybe this file should be untouched? (`session_status() === PHP_SESSION_ACTIVE`)